src/actions/bitcoinClient.ts

```diff
@@ -84,3 +84,22 @@ export const getRawTransaction = async (
   const result = await fetch(`${baseURL}/tx/${txid}`);
   return await result.json();
 };
+
+// Function to transmit a raw transaction to the network
+export const transmitRawTransaction = async (hex: string): Promise<any> => {
```

Comment:
Is this still required now that we can get Leather to correctly send the tx?

Reply:
This is needed to broadcast the signed psbt here https://github.com/stacks-network/sbtc-bridge/blob/6e81b487a3451721b7418149015c7c968d86087d/src/comps/ReclaimManager.tsx#L406